src/main/resources/hudson/plugins/ansicolor/Messages.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-DisplayName = Color ANSI Console Output
+plugin_DisplayName=Color ANSI Console Output

```

Comment:
I changed this because FindBugs was complaining about a method starting with a capital letter, so I prefixed it with the lowercase `plugin_`

I asked on the Jenkins IRC to see if there was a way to ignore generated because I am not familiar with the build process for Jenkins projects but didn't get an answer.